src/alttxt/parser.py

```diff
@@ -56,7 +56,62 @@ def load_data(self, file_path: Path) -> "dict[str, dict[str, Any]]":
         """
         with open(file_path) as f:
             return json.load(f)
-
+        
+    def classify_subset(self, degree: int, num_individual_sets: int) -> IntersectionType:
+        """
+        Classifies a subset based on its degree and the number of individual sets.
+        """
+        if degree == 1:
+            return IntersectionType.INDIVIDUAL
+        if degree == 0:
+            return IntersectionType.EMPTY
+        if degree == num_individual_sets:
+            return IntersectionType.ALL_SET
+        
+        if num_individual_sets == 3:
+            if degree == 2:
+                return IntersectionType.MEDIUM_SET
+            elif degree == 3:
+                return IntersectionType.HIGHORDER_SET
+            
+        elif num_individual_sets == 4:
+            if degree == 2:
+                return IntersectionType.LOW_SET
+            elif degree == 3:
+                return IntersectionType.MEDIUM_SET
+            elif degree == 4:
+                return IntersectionType.HIGHORDER_SET
+            
+        elif num_individual_sets == 5:
+            if degree == 2:
+                return IntersectionType.LOW_SET
+            elif degree == 3:
+                return IntersectionType.MEDIUM_SET
+            elif degree == 4:
+                return IntersectionType.HIGHORDER_SET
+            elif degree == 5:
+                return IntersectionType.HIGHORDER_SET
+        
+        elif num_individual_sets == 6:
+            if degree == 2:
+                return IntersectionType.LOW_SET
+            elif degree == 3:
+                return IntersectionType.MEDIUM_SET
+            elif degree == 4:
+                return IntersectionType.MEDIUM_SET
+            elif degree == 5:
+                return IntersectionType.HIGHORDER_SET
+            elif degree == 6:
+                return IntersectionType.HIGHORDER_SET
```

Comment:
This seems very repetitive. I'm sure there's a way to reduce this (ex: degree == 2 is always low_set)
At the very least combining values for repeated orders